Assets/Scripts/Entities/Enemy.cs

```diff
@@ -176,5 +179,17 @@ private void SetWorldPhase(EWorldPhase newPhase)
         {
             WorldPhase = newPhase;
         }
+
+        protected void NotifyAnimator(Vector3 movement)
```

Comment:
Definitely welcoming other ways to handle the animator being updated with the current move.